FFWEB-1248 be more specific about optional and required templates (#195)
FACT-Finder-Web-Components/documentation: markdown/3.x/en/ff-asn.md

<ff-asn-group-slider></ff-asn-group-slider>
```

> A slider can have a `[slot="groupCaption"]` attribute and a `[data-container="removeFilter"]` element but instead of `detailedLinks` and `hiddenLinks` the slider group accepts an `ff-slider-control`.

Reviewer: Is it mandatory to have both `[slot="groupCaption]` and `[data-container="removeFilter"]` in the same time to replace `detailedLinks` and `hiddenLinks` with `ff-slider-control`? I think they're both optional, but when reading this sentence I understood this like: 'you can replace 'detailedLinks' and 'hiddenLinks' with 'ff-slider-control' when both `[slot="groupCaption"]` attribute and `[data-container="removeFilter"]` are present'. Please, try to express this in a way that does not allow similar interpretations.

Author: As it is already stated above, that the slider works differently, I think mentioning elements which are not supported is only confusing. What do you think about [my suggestion below](https://github.com/FACT-Finder-Web-Components/documentation/pull/195/files#r294187097)?

> A slider can have a `[slot="groupCaption"]` attribute and a `[data-container="removeFilter"]` element but instead of `detailedLinks` and `hiddenLinks` the slider group accepts an `ff-slider-control`.

Author: ```suggestion
A slider can have a `[slot="groupCaption"]` attribute, a `[data-container="removeFilter"]` element and a `ff-slider-control` element.
```

Reviewer: I'm confused with `a` and `an` usage before `ff-*`. As far as I know we use `a` before words starting with consonants, `an` before words starting with vowels and whose pronounce like vowel-started (like "an hour"). Which is correct here: "a ff-component" or "an ff-component". The later one suits better for me.

Author: I am totally with you here 👍

> If no templates are provided for the `ff-slider-control` the templates default to the example below. If you want to customize the templates you must provide an `ff-slider` and an `input` element with attribute `data-control='1'` as well one with `data-control='2'`. 

Reviewer: ```suggestion
If no templates are provided for the `ff-slider-control`, the templates default to the example below. If you want to customize the templates, you must provide an `ff-slider` and an `input` element with attribute `data-control='1'` as well as one with `data-control='2'`. 
```

> The input elements automatically react to user input (enter key pressed) and start to filter. You can change the behavior of the controls by using the appropriate attributes as described in `ff-slider-control-section`'s [API documentation](/api/3.x/ff-asn#tab=api)

Reviewer: ```suggestion
The input elements automatically react to user input ('Enter' key pressed) and start filtering. You can change the behavior of the controls by using the appropriate attributes as described in `ff-slider-control` section's [API documentation](/api/3.x/ff-asn#tab=api)
```

    </div>

    <ff-slider-control submit-on-input="true">
       <div>
           <ff-slider step-size="1" submit-on-release="true">
               <div slot="slider1"></div>
               <div slot="slider2"></div>
           </ff-slider>
           <div slot="sliderControls" style="display: flex; justify-content: space-around; align-items: center">
               <input data-control="1" style="width: 60px;">
               <span style="width: 20px; height: 2px; background-color: black; display: inline-block"></span>
               <input data-control="2" style="width: 60px;">
            </div>

>        </div>

Reviewer: Was this additional `div` wrapper necessary to make it working? I remember there was some change in ff-slider-control related to this - just want to make sure why you changed the documentation here. And why added additional style to `sliderControls`? Did you just check if this example work on our controls? If so, it's OK then.

Author: yeah it worked before and works now as well.

Now the shown template is exactly the default template. I thought it is a nice easy extra information to write _"the templates default to the example below."_, as I added it.

Reviewer: Ah, I see now. I missed the context.
